filebeat/harvester/registry.go

```diff
@@ -39,9 +45,7 @@ func (r *Registry) Stop() {
 	close(r.done)
 
 	for _, hv := range r.harvesters {
-		r.wg.Add(1)
 		go func(h Harvester) {
-			r.wg.Done()
 			h.Stop()
 		}(hv)
 	}
```

Comment:
It is waiting for Completion in the defer statement on line 36. I quite like that Stop is actually blocking because someone from outside the harvester should not require knowlege about its inner working. I need to check what we do in other cases with `Stop()`